### PR TITLE
Korean Translation - Push dialog

### PR DESCRIPTION
**Pull request by progh2**
=)

### CODE REVIEW

File: src/translations/sqlb_ko_KR.ts [hunk 1]
```diff
@@ -4218,7 +4222,7 @@ Ctrl+Shift를 누른 상태에서 점프하고자 하는 곳을 클릭하세요<
     <message>
         <location filename="../VacuumDialog.ui" line="14"/>
         <source>Compact Database</source>
-        <translation>데이터베이스 크기 줄이기</translation>
+        <translation>데이터베이스 크기 줄이기(Vaccum)</translation>
```
Comment (justinclift) [inline src/translations/sqlb_ko_KR.ts]: Oops, spelling mistake here.  Vaccum → Vacuum

**Comment by justinclift** [comment]
Thanks.  There's one typo in it.  Hmmm, actually I can just fix that myself directly...

**Comment by justinclift** [comment]
@progh2 All done now.  I've merged both of those commits + a typo fix for two occurrences of Vaccum → Vacuum as 5ba7d5f9e2e86324267d612b129817e5974d4577.

Thanks for always putting time and effort into this. :smile:

**Comment by progh2** [comment]
Thanks. =D

**Pull request closed without merging**